restapi/stories.go

```diff
@@ -377,6 +377,20 @@ func (api api) CreateStories(w http.ResponseWriter, r *http.Request) render.Rend
 	return rest.MsgStatusOK("Stories added successfully")
 }
 
+func (api api) PublishStories(w http.ResponseWriter, r *http.Request) render.Renderer {
+	var stories []models.Story
+	if err := json.NewDecoder(r.Body).Decode(&stories); err != nil {
+		return rest.ErrInvalidRequest(api.logger, "Invalid payload", err)
+	}
+	for _, story := range stories {
```

Comment:
So just to be clear esentially all this is doing is updating each story object right? One thing to consider would be adding a check that only the `is_visible` field has changed but that might be a bit of work